bdio-protobuf/src/main/java/com/blackducksoftware/bdio/proto/BdioChunk.java

```diff
@@ -0,0 +1,87 @@
+/*
+ * Copyright (C) 2023 Synopsys Inc.
+ * http://www.synopsys.com/
+ * All rights reserved.
+ *
+ * This software is the confidential and proprietary information of
+ * Synopsys ("Confidential Information"). You shall not
+ * disclose such Confidential Information and shall use it only in
+ * accordance with the terms of the license agreement you entered into
+ * with Synopsys.
+ */
+package com.blackducksoftware.bdio.proto;
+
+import java.util.Collections;
+import java.util.Objects;
+import java.util.Set;
+
+import com.blackducksoftware.bdio.proto.v1.ProtoAnnotationNode;
+import com.blackducksoftware.bdio.proto.v1.ProtoComponentNode;
+import com.blackducksoftware.bdio.proto.v1.ProtoDependencyNode;
+import com.blackducksoftware.bdio.proto.v1.ProtoFileNode;
+import com.blackducksoftware.bdio.proto.v1.ProtoImageLayerNode;
+import com.blackducksoftware.bdio.proto.v1.ProtoImageNode;
+
+/**
+ * This class holds data deserialized from bdio chunk in protobuf format
+ * 
+ * @author sharapov
+ *
+ */
+public class BdioChunk {
+
+    private final Set<ProtoFileNode> fileNodes;
```

Comment:
Nitpick: All other files in this MR are using tabs, this one uses spaces. Which one is intended?